### PR TITLE
Make status reasons Camel-Case

### PR DESCRIPTION
**Pull request by takebayashi**
Camel-Case-style status reasons are generally preferred.

### CODE REVIEW

File: Sources/Status.swift [hunk 1]
```diff
@@ -62,123 +62,123 @@ public enum Status : Int, CustomStringConvertible {
...
+      return "Request Entity Too Large"
     case .RequestURITooLong:
-      return "REQUEST-URI TOO LONG"
+      return "Request-URI Too Long"
     case .UnsupportedMediaType:
-      return "UNSUPPORTED MEDIA TYPE"
+      return "Unsupported Media Type"
     case .RequestedRangeNotSatisfiable:
-      return "REQUESTED RANGE NOT SATISFIABLE"
+      return "Requested Range Not Satisfiable"
```
Comment (kylef) [inline Sources/Status.swift]: This one is a full sentence and could be mostly lower-cased as it is in RFC2616:

```
| "416"  ; Section 10.4.17: Requested range not satisfiable
```

Comment (takebayashi) [inline Sources/Status.swift]: Fixed it.

**Comment by kylef** [comment]
Looks like the [tests are failing](https://travis-ci.org/nestproject/Inquiline/builds/111063739) and need to be [updated for these changes](https://github.com/nestproject/Inquiline/blob/master/Specs/StatusSpec.swift#L12).

Would you use able to update them and squash your commits? Then this is good to go.

**Comment by takebayashi** [comment]
I fixed the tests. Now all tests are green.

**Comment by kylef** [comment]
Thanks @takebayashi!

**Pull request merged**